<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="shopWidget">
   <widget class="QPushButton" name="exploreWilds">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Explore
Wilds
(d)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopPokemon1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>680</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopPokemon2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>680</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopPokemon3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>680</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopPokemon4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>680</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shopPokemon0">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>680</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pokemon 0</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="timeToNextStage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>981</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="teamMember0">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Battle Pokemon 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Your Upcoming Team</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Your Upcoming Opponent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon5">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>520</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teamMember1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Battle Pokemon 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="teamMember2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Battle Pokemon 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeTeamMember0">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeTeamMember1">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeTeamMember2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam0Up">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam1Up">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam2Up">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam0Down">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>70</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam1Down">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shiftTeam2Down">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>210</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Party</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="opposingPokemon0">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Opposing Pokemon 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="opponentName">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Insert Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="shopLocationLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>640</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Location Label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty0">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon0">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems0">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyItems2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Items</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="partyPokemon1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Team Pokemon 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addParty5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageStorage">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Storage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pokeBallCount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="opponentName_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>511</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Poke Balls</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="energyCount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>549</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="opponentName_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Energy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="hitPoints">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>529</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="opponentName_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hit Points</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
